docs/content/guides/columns/react-hot-column.md

```diff
@@ -158,7 +158,7 @@ class UnconnectedColorPicker extends BaseEditorComponent {
   constructor(props) {
     super(props);
 
-    this.editorRef = React.createRef();
+    this.editorRef = React.createRef(null);
```

Comment:
Could it be changed with `useRef(null)`?

Reply:
hey, @wszymanski - I checked it now once again, but unfortunatelly `useRef(null)` works correctly only in functional components and in this case your proposition won't work.

![useref](https://user-images.githubusercontent.com/112617879/191945716-2726c537-70e1-4586-8a0e-199eff8826a0.jpg)